README.md

```diff
@@ -39,37 +39,47 @@ In your React project:
 import { ComponentName } from '@openedx/paragon';
 ```
 
-#### SCSS Foundation
+#### CSS Foundation
 
-Usage for Open edX and others:
+**Usage with no theme:**
 
-**index.scss**
+The Paragon CSS Foundation offers a collection of fundamental styles, encompassing padding, typography, and sizing.
+When applied, these styles contribute to the creation of a straightforward and neat appearance for your application.
+
+To integrate these foundational styles into your project, include the following import:
 
 ```
-// ... Any custom SCSS variables should be defined here
-@import '~@openedx/paragon/styles/scss/core/core.scss';
+@use "@openedx/paragon/dist/core.min.css";
 ```
 
-Usage on with `@edx/brand`:
+**Usage with a theme:**
+
+In addition to the basic styles, you can provide a theme layer that includes styles for colors, shadows, backgrounds
+and transparency of elements. This allows you to visually represent a specific theme in your application.
 
-**index.scss**
+To apply these theme-specific styles, include the following import:
 
 ```
-@import '~@edx/brand/paragon/fonts.scss';
-@import '~@edx/brand/paragon/variables.scss';
-@import '~@openedx/paragon/styles/scss/core/core.scss';
-@import '~@edx/brand/paragon/overrides.scss';
+@use "@my-brand/light.min.css";
 ```
```

Comment:
I feel like this section is a bit ambiguous. It's not clear if someone wanting to use a theme should have

```
@use "@openedx/paragon/dist/core.min.css";
@use "@my-brand/light.min.css";
```

or just 

```
@use "@my-brand/light.min.css";
```

Reply:
Yes, I agree
The first variant will be more understandable
Thank you!